packages/cli/docs/cli.md

```diff
@@ -114,11 +114,14 @@ If you re-run this command on an existing directory it will leave existing files
 You'll need to do a `zapier push` before the new version is visible in the editor, but otherwise you're good to go.
 
 **Arguments**
-* (required) `integrationId` | To get the integration/app ID, go to "https://developer.zapier.com", click on an integration, and copy the number directly after "/app/" in the URL.
 * (required) `path` | Relative to your current path - IE: `.` for current directory.
 
 **Flags**
-* (required) `-v, --version` | Convert a specific version. Required when converting a Visual Builder integration.
+* `-i, --integrationId` | To get the integration/app ID, go to "https://developer.zapier.com", click on an integration, and copy the number directly after "/app/" in the URL.
+* `-v, --version` | Convert a specific version. Required when converting a Visual Builder integration.
+* `-j, --json` | The JSON definition to use, as alternative for reading from a Visual Builder integration. Must be a JSON-encoded object. The data can be passed from the command directly like '{"key": "value"}', read from a file like @file.json, or read from stdin like @-.
```

Comment:
Could we add an example of what the definition JSON should look like below the flags, to make it easier for external users to take advantage of this? If it'll be quite long we could always put it in a [collapsed section](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections).